apps/molgenis-components/src/components/forms/InputString.vue

```diff
@@ -18,7 +18,7 @@
         @input="$emit('input', $event.target.value)"
         type="text"
         class="form-control"
-        :class="{ 'is-invalid': errorMessage }"
+        :class="{ 'is-invalid': stringError }"
```

Comment:
what happend to the errorMessage , maybe ````stringError || errorMEssage```` ? or  maybe i'm missing something

Reply:
stringError is a computed that gives the specific string error or the more global errorMessage. See line 52 and 54.